tests/performance_tests/performance_test_basic.py

```diff
@@ -94,17 +94,20 @@ def argsStr(self) -> str:
         maximumClients: int = 0
         loggingDict: dict = field(default_factory=lambda: { "bios": "off" })
         prodsEnableTraceApi: bool = False
+        isNodeosv2: bool = False
         specificExtraNodeosArgs: dict = field(default_factory=dict)
         _totalNodes: int = 2
 
         def __post_init__(self):
             self._totalNodes = self.pnodes + 1 if self.totalNodes <= self.pnodes else self.totalNodes
             if not self.prodsEnableTraceApi:
                 self.specificExtraNodeosArgs.update({f"{node}" : "--plugin eosio::trace_api_plugin" for node in range(self.pnodes, self._totalNodes)})
+            if self.isNodeosv2:
+                self.specificExtraNodeosArgs.update({f"{node}" : "--plugin eosio::history_api_plugin --filter-on \"*\"" for node in range(self.pnodes, self._totalNodes)})
```

Comment:
See @jgiszczak's comment above: `Python allows the use of the ' character for string quoting to avoid having to escape quotes. If a string contains double quotes, the enclosing quotes can be single quotes and no blackslashes are necessary.`